contracts/erc7683/ERC7683Depositor.sol

```diff
@@ -79,7 +79,7 @@ abstract contract ERC7683OrderDepositor is ISettlementContract {
             // Note: simplifying assumption to avoid quote timestamps that cause orders to expire before the deadline.
             SafeCast.toUint32(order.initiateDeadline - QUOTE_BEFORE_DEADLINE),
             order.fillDeadline,
-            acrossOrderData.exclusivityDeadline,
+            SafeCast.toUint32(getCurrentTime()) + acrossOrderData.exclusivityDeadlineOffset,
```

Comment:
Is it worth applying the cast directly in `getCurrentTime()`, so it returns uint32 instead? It seems like we'd normally want to trim it down to a uint32 anyway.